guides/security/device_admin_profiles.md

```diff
@@ -0,0 +1,31 @@
+# Admin profiles and standard user profiles on devices
+
+As part of our commitment to secure working practices, we install separate admin and standard user profiles on laptops. The reason this restriction is in place is to prevent the accidental downlaoding and installation of malicious software on company devices, to close an attack vector, and to protect both Made Tech and our customers.
+
+Because of our heavy use of cloud-based software, we do not believe removing default admin access will prevent people from working productively. If a team member can provide a business case for why they need to retain admin access to their device as default, the ISMS governance team will review this policy.
+
+## Business Teams
+
+Separate admin and standard profiles must be installed on devices being used by members of business teams at Made Tech (Sales, People, Operations etc.), who do not engage in engineering and active development.
+
+Your device will be set up with separate admin and standard profiles as standard. You will be given access to the admin profile so you can install any necessary software.
+
+If you are a member of a business team, and using a device with split admin and standard user profiles, you should follow these guidelines:
+ - You should always log into your standard user profile as default.
+ - You can download approved software from the list below using your admin credentials, or any software necessary for your regular work.
+ - You should use your admin credentials to install any software or OS updates on a regular basis to ensure you are always running the latest version. 
+ - If you are uncertain about what you are downloading and installing, please talk to the CSO or ISMS manager for support. 
+
+
+## Engineering and Delivery Teams
+
+We have agreed a clear business case for people in engineering roles to use an admin profile on their device as default because of the need to install packages related to software we're developing on an almost daily basis.
```

Comment:
It's worth mentioning that even though this is "admin by default", you are still prompted for authentication whenever you utilise admin capabilities.

This is the case with MacOS and Linux